worker/rollback-handler/.eslintrc.yml

```diff
@@ -0,0 +1,14 @@
+env:
+  es2021: true
+  node: true
+parser: "@babel/eslint-parser"
```

Comment:
this should not be required and can be removed

Reply:
done with [548ca79](https://github.com/klips-project/klips-sdi/pull/131/commits/548ca79f0a2ffa74f3c33ad64e19f9a376258cdd)